pkg/agent/flowexporter/connections/connections.go

```diff
@@ -142,6 +150,50 @@ func lookupServiceProtocol(protoID uint8) (corev1.Protocol, error) {
 	return serviceProto, nil
 }
 
+func (cs *connectionStore) addNetworkPolicyMetadata(conn *connection.Connection) {
+	// Retrieve NetworkPolicy Name and Namespace by using the ingress and egress
+	// IDs stored in the connection label.
+	if len(conn.Labels) != 0 {
+		if klog.V(4).Enabled() {
+			klog.InfoS("Setting NetworkPolicy metadata from connection labels", "labels", hex.EncodeToString(conn.Labels))
+		}
+		ingressOfID := binary.LittleEndian.Uint32(conn.Labels[:4])
```

Comment:
Do you think `binary.LittleEndian` is the reason that we need a reverse when parsing the values from ct_label? libOpenflow always uses binary.BigEndian in `MarshalBinary` functions?

Reply:
It's not the reason, but it shows the discrepancy between libOpenflow (always BigEndian) and the conntrack library (NativeEndian, which on most systems means LittleEndian). I have cleaned this up, see https://github.com/antrea-io/antrea/pull/7361#discussion_r2305348611